document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoQueryExecutorIntegrationTest.java

```diff
@@ -596,6 +598,34 @@ public void testFilterAndUnnest() throws IOException {
     assertSizeEqual(query, "mongo/unwind_filter_response.json");
   }
 
+  @Test
+  public void testQueryQ1AggregationFilterAlongWithNonAliasFields() throws IOException {
```

Comment:
As currently, we are not supporting other than the alias field (or aggregation / functional), moved this test here from `DocStoreTest`.
So, we know what is not compatible and needs work.